cluster-autoscaler/cloudprovider/clusterapi/clusterapi_controller_test.go

```diff
@@ -73,6 +73,14 @@ const customCAPIGroup = "custom.x-k8s.io"
 const fifteenSecondDuration = time.Second * 15
```

Comment:
do we really need to carry this commit? can't this test case be covered upstream?

Reply:
I think you're right, apart from the changes in [clusterapi_utils_test.go](https://github.com/openshift/kubernetes-autoscaler/pull/231/commits/5c3ac70ed793f921453516ab6bad0db0b9f1c82a#diff-44f848f51d0ba33c7696002d371d4549b0daa7d8e6dbf173cfda12b8f125b0c9), we are not really adding anything. We will still need that small fix for the clusterapi_utils test though otherwise our other changes which set the group to machine.openshift.io break the test